sql/session.go

```diff
@@ -695,15 +698,13 @@ func (c *Context) QueryTime() time.Time {
 // children of this span.
 func (c *Context) Span(
```

Comment:
might also consider splitting this method into `Span(opName string)` and `SpanWithOption(opName string, option trace.SpanStartOption` to remove the variadic parameter allocation

Reply:
No allocation when we don't pass anything. We would save the spilling and filling the two registers I guess...Doubt it is worth it for now?